airbyte-cdk/python/README.md

```diff
@@ -130,6 +130,27 @@ To run acceptance tests for multiple connectors using the local CDK, from the ro
 ./airbyte-cdk/python/bin/run-cats-with-local-cdk.sh -c <connector1>,<connector2>,...
 ```
 
+#### When you don't have access to the API
+There can be some time where you do not have access to the API (either because you don't have the credentials, network access, etc...) You will probably still want to do end-to-end testing at least once. In order to do so, you can emulate the server you would be reaching using a server stubbing tool.
+
+For example, using `mockserver`, you can set up an expectation file like this:
```

Comment:
nit, might be worth a link to the documentation site about how to use this tool for more detail